lib/services/servermanagement/generate.cmd

```diff
@@ -0,0 +1,17 @@
+::

```

Comment:
please close this PR and send a PR to the master branch. You would be better off copying over the code to master. 
To give some context: Initially I was told to manage a separate branch for autorest. Hence, I deleted all the stuff that wasn't autorest specific. Later it was decided to have everything in master. I wasn't able to merge autorest into master as that will delete all the non autorest specific stuff in master. Hence, I ended up copying over everything to master in my fork and then sent a PR. 

Sorry for all the confusion. I shall be deleting this branch in the next week to avoid any confusion.

One more thing, please have a README.md, package.json and License.txt in your sericemanagement folder. You will be publishing to npm using a Jenkins ci job. We don't ship rollup packages anymore.

All of this is covered in the onboarding doc link I mentioned in the previous comment.